### ISSUE TITLE
[BUG] Unable to send images or media files in Desktop App

### ISSUE THREAD

**Issue by alexander-mahisoft** [oxen-io/session-desktop#2943]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-desktop/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I've been unable to send any kind of media files or attachments on private chats or privately created rooms from the Desktop or iOS app since a few days ago.

Media messages get the loading icon for a few seconds and then show "Failed to send"

I'm attaching the Debug log from the Desktop version

### Steps To Reproduce

In a private chat with another user or a private group, send any attachment file

### Desktop Version

v1.11.3

### Anything else?

Debug Log:
[session_debug_1696368503151.txt](https://github.com/oxen-io/session-desktop/files/12797078/session_debug_1696368503151.txt)

Example screenshot:
![image](https://github.com/oxen-io/session-desktop/assets/84341320/c52f51cd-c7b6-489f-81e3-31e61193e00d)

**Comment by KeeJef** [comment]
Hey, are you still having issues uploading attachments, we did have an issue with the file server for a few days, but should now be resolved

**Comment by Dikowda** [comment]
Urgent:  Same issue for me today...

ERROR 2023-12-16T14:01:35.505Z Message.saveErrors: null Error: upload to file server v2 of <SOME FILE> failed
    at uploadToFileServer (C:\Program Files\Session\resources\app.asar\ts\session\utils\Attachments.js:74:11)
    at async Promise.all (index 0)
    at async Promise.all (index 0)
    at async MessageModel.uploadData (C:\Program Files\Session\resources\app.asar\ts\models\message.js:525:47)
    at async MessageModel.retrySend (C:\Program Files\Session\resources\app.asar\ts\models\message.js:584:74)
    at async C:\Program Files\Session\resources\app.asar\ts\components\conversation\message\message-content\MessageContextMenu.js:119:13

I have tried multiple file types and all fail.  This makes Session unusable!

**Comment by sysfu** [comment]
I experience this issue frequently with media messages (primarily voicemails in my case).  To a lesser extent it also happens with plain old text message.

Today I've tried resending a voicemail six times and it's failed on each attempt. Most of the time I just give up and try later in the day or another day entirely. Really bad for usability, I can't recommend this app to my contacts when this happens regularly.

Logs attached

[session_debug_1704470325425.txt](https://github.com/oxen-io/session-desktop/files/13844045/session_debug_1704470325425.txt)

**Comment by Dikowda** [comment]
The lack of response form the devs and the other issues with the app namely unreliable group management (for example user appears to be removed from a group but they are not), means I've given up using it and migrated to one that can do the basics.

**Comment by alexander-mahisoft** [comment]
@Dikowda yep seems it's a recurring issue, have the same thing happening right now so it must be server side

**Comment by hydrian** [comment]
I'm still getting this error in Session Desktop with version 1.11.5. I'm getting "There was an Error setting this attachment." I'm trying to send a 150k jpeg image. This is with Linux Mint 21.2 and the Flathub installation.

**Comment by KeeJef** [comment]
> I'm still getting this error in Session Desktop with version 1.11.5. I'm getting "There was an Error setting this attachment." I'm trying to send a 150k jpeg image. This is with Linux Mint 21.2 and the Flathub installation.

Do you have logs from the client around the time you ran into the issue, is it still reproducible?

**Comment by hydrian** [comment]
> > I'm still getting this error in Session Desktop with version 1.11.5. I'm getting "There was an Error setting this attachment." I'm trying to send a 150k jpeg image. This is with Linux Mint 21.2 and the Flathub installation.
> 
> Do you have logs from the client around the time you ran into the issue, is it still reproducible? 

It happens with any image file all the time. 

I'll attach the logs the next time I get in front of my desktop.

**Comment by hydrian** [comment]
I got two different errors today. 

1.  The issue presented as the image wouldn't send even after multiple resends. This issue creates log entries to diagnose. I've attached that debug log excerpt and the image I was uploading. 
[debuglog-wont_send.txt](https://github.com/oxen-io/session-desktop/files/14184398/debuglog-wont_send.txt)
![Stag-Phases](https://github.com/oxen-io/session-desktop/assets/988950/5846cc6e-0801-4b4f-867c-7e64baded435)
1. The second issue didn't crate any log entries. So I'm at a loss at what I can give you. Here is a screenshot of the error. 
![Session-WontSet-2024-02-06_13-18](https://github.com/oxen-io/session-desktop/assets/988950/858c54f5-403c-4926-a518-31144cd86f47)

Here is the flathub application info: 
Session Desktop - Private messenger

          ID: network.loki.Session
         Ref: app/network.loki.Session/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.11.5
     License: GPL-3.0
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 332.8 MB
     Runtime: org.freedesktop.Platform/x86_64/22.08
         Sdk: org.freedesktop.Sdk/x86_64/22.08

      Commit: 8aad381e1148cd6d071cbfec04e6958cec7072f6d5f068525ca8f950e443bb04
      Parent: 14b23301cf78b274523d3ebff508df0b66ce039e31bb5e251c9ea5a0fa29ee03
     Subject: Update session-desktop-linux-amd64-1.11.4.deb to 1.11.5 (8099bb55)
        Date: 2023-12-11 03:27:22 +0000

**Comment by KeeJef** [comment]
Tracking this issue internally via https://optf.atlassian.net/browse/SES-1454

**Comment by hydrian** [comment]
Upgraded to 1.11.5. This didn't help. I didn't expect it to help based on the changelog. Wishful thinking. 

I did try the apt deb package version of 1.11.5 and it didn't give me the constant 'Sorry, there was an error setting your attachment.' error message when I try to attach a file unlike the flatpak version.

**Comment by sysfu** [comment]
> I experience this issue frequently with media messages (primarily voicemails in my case). To a lesser extent it also happens with plain old text message.
> 
> Today I've tried resending a voicemail six times and it's failed on each attempt. Most of the time I just give up and try later in the day or another day entirely. Really bad for usability, I can't recommend this app to my contacts when this happens regularly.

I'm pleased to report that since the latest app update to 1.12.0 my deliver-ability problems have almost entirely disappeared.

**Comment by hydrian** [comment]
> 
> I'm pleased to report that since the latest app update to 1.12.0 my deliver-ability problems have almost entirely disappeared.

What install type did you use?

**Comment by sysfu** [comment]
> > I'm pleased to report that since the latest app update to 1.12.0 my deliver-ability problems have almost entirely disappeared.
> 
> @sysfu, What install type did you use?

AppImage

**Comment by sysfu** [comment]
The issue has re-appeared on occasion with 1.12.4 release. Some messages, even just basic text ones have to be resent 4-5 Xs before they go through.

Perhaps this is due to the recent spam attacks on the network.

**Comment by KeeJef** [comment]
Is this still occurring in 1.13.1?

**Comment by sysfu** [comment]
Sorry for delayed response on this.

I'm having a similar failed to send issue with a message that includes a URL preview as of release v1.13.2. 

Not sure if it's the same bug.

[session_debug_2024-09-18T19_31_13.575Z.txt](https://github.com/user-attachments/files/17049368/session_debug_2024-09-18T19_31_13.575Z.txt)

**Comment by sysfu** [comment]
Did it again today with a jpg image

[session_debug_2024-09-19T21_01_54.624Z.txt](https://github.com/user-attachments/files/17066633/session_debug_2024-09-19T21_01_54.624Z.txt)

**Comment by sysfu** [comment]
I just learned that the group chat I was trying to send images too has banned uploading of images. They were subject to earlier attacks of bad actors uploading CSAM.

I just tried sending the image in a direct message and it went through.
